CHANGELOG.md

```diff
@@ -8,6 +8,10 @@
 - 🐛 Fix: Only use major version in assembly version (ex: `12.0.0.0`, instead of
   `12.0.1.0`) ([#18][#18])
 
+- 🐛 Fix: Package targets .NET 4.5 & .NET 4.6.1 instead of recommended
+  .NET Standard 2.0. Portable should still target the [PCL profile 259
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [[no-undefined-references] Found reference to undefined definition](https://app.codacy.com/manual/jilleJr/Newtonsoft.Json-for-Unity/pullRequest?prid=4743159)